app/views/profiles/show.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= page_title "Profile" %>
```

Comment:
When the profile is from the current user, I think this title is ok. But if it's a public profile from somebody else, I think the title should contain the person's name i.e "John Doe's Profile"

Reply:
@LuigiR0jas good eye, I set the page title to the user's name regardless of whether the profile belongs to the current user or not, I think it's more consistent but I'm open to implementing your suggestion.